src/components/Navbar.jsx

```diff
@@ -78,15 +79,22 @@ const Navbar = () => {
     : "";
 
   const getZoomBalance = async () => {
-    const balance = await contracts.ZoomContract.balanceOf(state.wallet);
+    const balance = await contracts.ZoomContract.balanceOf(wallet);
     setZoomBalance(balance / 1000000000000000000);
```

Comment:
We may want to start always using https://docs.ethers.io/v5/api/utils/bignumber/  if it works..  I had some funky big number issues in zoombies.. web3.js  but ethers seems way better for this